bundler.go

```diff
@@ -396,10 +404,23 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 		gp = build.Default.GOPATH
 	}
 
+	args := []string{"build", "-ldflags", std.String()}
+	var flag string
+	for k, v := range b.buildFlags {
+		if hasDash := strings.HasPrefix(k, "-"); hasDash {
+			flag = k
+		} else {
+			flag = "-" + k
+		}
+		args = append(args, flag, v)
```

Comment:
People may need to use `boolean` flags (without value) therefore I would only append `v` if `!= ""`

Reply:
Good point, I have made the change.